store/sqlstore/store.go

```diff
@@ -7,7 +7,6 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	_ "github.com/lib/pq"
 	"github.com/mattermost/gorp"
-
```

Comment:
nit: usually the paradigm is to break up imports into "stdlib", "external" and "internal"